opal/mca/accelerator/accelerator.h

```diff
@@ -308,7 +308,7 @@ typedef struct {
  *
  * @return                   Initialized module or NULL if init failed.
  */
-typedef opal_accelerator_base_module_t * (*mca_accelerator_base_component_init_fn_t)();
+typedef opal_accelerator_base_module_t * (*mca_accelerator_base_component_init_fn_t)(void);
```

Comment:
This should be part of another commit

Reply:
done

Reply:
I think you misunderstood what I meant. I meant this should be part of https://github.com/wckzhang/ompi/pull/3/commits/35e50fe19cd101d30a58c3664cd3b4d1f4ce947a which is where you removed int *priority